MapboxDirections/MBRouteOptions.swift

```diff
@@ -304,6 +310,13 @@ open class RouteOptions: NSObject, NSSecureCoding {
     open var includesExitRoundaboutManeuver = false
     
     /**
+     If set, this `Locale` will be used to translate instructions.
+     
+     See supported languages [here](https://www.mapbox.com/api-documentation/#instructions-languages).
+     */
+    open var locale: Locale?
```

Comment:
It’d be nice to default to `Locale.autoupdatingCurrent` or `Bundle.main.preferredLocalizations.first` so the behavior is always what the user would expect, but that can happen as tail work, possibly at the same time that we set a default locale [in MapboxGeocoder.swift](https://github.com/mapbox/MapboxGeocoder.swift/blob/631481f2e01cb9151b7211895cb6a5842d6081d2/MapboxGeocoder/MBGeocodeOptions.swift#L56-L65).

Reply:
Let's keep this optional for now.